windows/win_nssm.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2015, Heyo
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# this is a windows documentation stub.  actual code lives in the .ps1
+# file of the same name
+
+DOCUMENTATION = '''
+---
+module: win_nssm
+version_added: "2.0"
+short_description: NSSM - the Non-Sucking Service Manager
+description:
+    - nssm is a service helper which doesn't suck. See https://nssm.cc/ for more information.
+requirements:
+  - "nssm >= 2.24.0 # (install via win_chocolatey) win_chocolatey: name=nssm"
+options:
+  name:
+    description:
+      - Name of the service to operate on
+    required: true
+    default: null
+    aliases: []
+
+  state:
+    description:
+      - State of the service on the system
+    required: false
+    choices:
+      - started
+      - stopped
+      - restarted
+      - absent
+    default: started
+    aliases: []
+
+  application:
+    description:
+      - The application binary to run as a service
```

Comment:
Since required is false and default is null, it would be nice to expand the description to mention that is a parameter to use when the service is already installed (I assume)